holoviews/util/settings.py

```diff
@@ -361,7 +361,7 @@ def update_options(cls, options, items):
     @classmethod
     def initialize(cls, backend_list):
         cls.backend_list = backend_list
-        backend = cls.options.get('backend', Store.current_backend)
+        backend = Store.current_backend
```

Comment:
> I'd strongly suggest you make a notebook and then switch back and forth between backends with output a few times, rerun the notebook_extension, set fig formats then switch backends and so on.

I've just done that (included rerunning ``notebook_extension``) and it seems to be working correctly.